src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/ReportRepositoryTests.kt

```diff
@@ -122,7 +123,7 @@ class ReportTests : CleanDatabaseTests()
 
         insertReport("report", "v3")
 
-        val sut = createSut(isReviewer = true)
+        val sut = Orderly(isReviewer = true, isGlobalReader = true, reportReadingScopes = listOf())
```

Comment:
Feels like it would be worth splitting out the tests for the residual Orderly class so they're separate from the ReportRepository tests. Maybe retain ReportTests.kt and keep for those Orderly test?

Reply:
Cool, actually all those Orderly tests are related to versions, so I've put them in the remaining `VersionTests` class.